manifests/c/CyberHive/Connect/3.8.1/CyberHive.Connect.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created with komac v2.12.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: CyberHive.Connect
+PackageVersion: 3.8.1
+InstallerLocale: en-US
+InstallerType: inno
+Scope: machine
+InstallerSwitches:
+  Custom: /MANAGED=1
+UpgradeBehavior: install
+ProductCode: '{EA126048-51DD-4E62-A524-9FA0C8F07C24}_is1'
+ReleaseDate: 2025-08-28
+AppsAndFeaturesEntries:
+- ProductCode: '{EA126048-51DD-4E62-A524-9FA0C8F07C24}_is1'
+ElevationRequirement: elevatesSelf
+InstallationMetadata:
+  DefaultInstallLocation: '%ProgramFiles%\CyberHive Connect'
+Installers:
+- Architecture: x86

```

- https://github.com/microsoft/winget-pkgs/blob/4bae5256427f0ed757c4d23389a3cd4b8537e418/manifests/c/CyberHive/Connect/3.8.0/CyberHive.Connect.installer.yaml#L21

```suggestion
- Architecture: x64
```